e2e-tests/portal/cypress/e2e/journeys/maker-bond/bond-form-values.js

```diff
@@ -1,11 +1,12 @@
-const { roundNumber } = require('../../../../../../portal-api/src/utils/number');
-const { oneMonth, today } = require('../../../../../e2e-fixtures/dateConstants');
+import { BOND_FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { roundNumber } from '../../../../../../portal-api/src/utils/number';
```

Comment:
Yes, it's strange. I didn't make this change, just changed to an `import` instead of `require`.

I have copied the function from portal-api into this E2E function.